DQMOffline/L1Trigger/interface/L1TTauOffline.h

```diff
@@ -0,0 +1,264 @@
+#ifndef DQMOFFLINE_L1TRIGGER_L1TTAUOFFLINE_H
+#define DQMOFFLINE_L1TRIGGER_L1TTAUOFFLINE_H
+
+#include "DataFormats/L1Trigger/interface/Muon.h"		 		
+#include "DataFormats/L1Trigger/interface/BXVector.h"	
+#include "DataFormats/L1Trigger/interface/Tau.h"
+#include "DataFormats/TauReco/interface/PFTauFwd.h"
+#include "DataFormats/TauReco/interface/PFTau.h"
+#include "DataFormats/TauReco/interface/PFTauDiscriminator.h"
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/LuminosityBlock.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "DQMServices/Core/interface/MonitorElement.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "DataFormats/L1GlobalMuonTrigger/interface/L1MuGMTReadoutCollection.h"
+#include "DataFormats/MuonReco/interface/Muon.h"
+#include "DataFormats/MuonReco/interface/MuonFwd.h"
+
+#include "TrackingTools/TransientTrack/interface/TransientTrack.h"
```

Comment:
hi @davignon - sorry for my second step of review - please clean up all the unneeded include files from this header (anything you don't use in the header should not be here.)